src/app/stylesheets/custom/accordion.less

```diff
@@ -35,7 +35,7 @@
 
   .panel-group .panel .panel-heading {
     padding: 0;
-    background-color: @dashboard-account-panel-bg-color;
+    background-color: @dashboard-account-tabs-bg-color;
```

Comment:
`@dashboard-account-panel-bg-color` is the right name to me. If you need to change the color of this variable, do it in the variable.less.